internal/controller/user/user.go

```diff
@@ -157,40 +158,33 @@ func (c *external) Observe(ctx context.Context, mg resource.Managed) (managed.Ex
 		return managed.ExternalObservation{}, nil
 	}
 
-	// TODO: GET User instead of listing users for group
-	users, err := c.cloudianService.ListUsers(ctx, group, nil)
+	_, err := c.cloudianService.GetUser(ctx, cloudian.User{GroupID: group, UserID: externalName})
 	if err != nil {
-		return managed.ExternalObservation{}, errors.Wrap(err, errListUsers)
+		if errors.Is(err, cloudian.ErrNotFound) {
+			return managed.ExternalObservation{ResourceExists: false}, nil
+		}
```

Comment:
`error.Is` is nil-safe, meaning it can be used outside of (before) the nil check.

I have no strong preference other than it would be nice to use the same pattern here and in https://github.com/statnett/provider-cloudian/blob/aae9cb86e107c3ef75c2192bb48673e7d5d51f39/internal/controller/accesskey/accesskey.go#L153-L159

Sort of viewing the sdk wrapping as (humanly) _generated_ code that is equal everywhere.

Reply:
Done. Made consistent now.